server/models/User.js

```diff
@@ -1,69 +1,85 @@
-const mongoose = require("mongoose");
-const bcrypt = require("bcryptjs");
+const mongoose = require('mongoose');
+const bcrypt = require('bcryptjs');
 const Schema = mongoose.Schema;
 
 // Create Schema
-const UserSchema = new Schema({
-  firstName: {
-    type: String,
-    required: true,
+const UserSchema = new Schema(
+  {
+    firstName: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    email: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    password: {
+      type: String,
+      required: true,
+    },
   },
-  lastName: {
-    type: String,
-    required: true,
-  },
-  email: {
-    type: String,
-    required: true,
-  },
-  password: {
-    type: String,
-    required: true,
-  },
-});
+  {
+    toObject: {
+      transform: function (doc, ret) {
+        delete ret.password;
+      },
+    },
+    toJSON: {
+      transform: function (doc, ret) {
+        delete ret.password;
+      },
```

Comment:
Nice work, we can use this later if we want to modify or delete something else when shown